list.go

```diff
@@ -50,3 +50,13 @@ type ListLookUpByIDResponse struct {
 	Detail   string              `json:"detail,omitempty"`
 	Type     string              `json:"type,omitempty"`
 }
+
+type ListsTweetsLookUpByIDResponse struct {
+	Tweets   []*Tweet            `json:"data"`
+	Includes *ListIncludes       `json:"includes,omitempty"`
```

Comment:
`Includes` field has `Tweet` field.
Please run
`curl "https://api.twitter.com/2/lists/84839422/tweets?expansions=referenced_tweets.id&tweet.fields=id,source,text,withheld&max_results=10" -H "Authorization: Bearer $token" `

FYI
https://developer.twitter.com/en/docs/twitter-api/lists/list-tweets/api-reference/get-lists-id-tweets#cURL

Reply:
Sorry, I missed it.